Sources/AblyAssetTrackingPublisher/Services/DefaultLocationService.swift

```diff
@@ -48,6 +48,7 @@ extension DefaultLocationService: PassiveLocationManagerDelegate {
     }
     
     func passiveLocationManager(_ manager: PassiveLocationManager, didUpdateLocation location: CLLocation, rawLocation: CLLocation) {
+        delegate?.locationService(sender: self, didUpdateRawLocationUpdate: RawLocationUpdate(location: rawLocation.toLocation()))
```

Comment:
What if raw locations are turned off?

Reply:
delegation works no matter if `raw locations` are enabled or not